rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -112,7 +112,7 @@ def create_data(self, context, data):
 
```

Comment:
For above code, a simple way to do is using the compact function from pydash.
```python
no_of_rmds = len(compact(rmd_list))
```

Reply:
Great point. Addressed.